examples/default.yaml

```diff
@@ -196,6 +196,13 @@ containerd:
 #     vim was not installed in the guest. Make sure the package system is working correctly.
 #     Also see "/var/log/cloud-init-output.log" in the guest.
 
+# Adds an additional archive to the CIDATA image which is mounted on boot
```

Comment:
Needs more explanation (how to access the archive from the provisioning scripts?)